packages/fxa-content-server/server/config/local.json-dist

```diff
@@ -22,10 +22,17 @@
     "validation": {
       "https://identity.mozilla.com/apps/oldsync": {
         "redirectUris": [
-         "http://localhost:3030/oauth/success/a2270f727f45f648",
-         "http://localhost:3030/oauth/success/1b1a3e44c54fbb58",
-         "urn:ietf:wg:oauth:2.0:oob:pair-auth-webchannel",
-         "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel"
+          "http://localhost:3030/oauth/success/a2270f727f45f648",
+          "http://localhost:3030/oauth/success/1b1a3e44c54fbb58",
+          "urn:ietf:wg:oauth:2.0:oob:pair-auth-webchannel",
+          "urn:ietf:wg:oauth:2.0:oob:oauth-redirect-webchannel",
+          "urn:ietf:wg:oauth:2.0:oob"
```

Comment:
Things didn't seem to work right without this URI. It is the listed redirect URI for Firefox, so why isn't it here?

I added both for Thunderbird.